scripts/two_line_bash.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+#
+#
+# This was casuing issues in tmux and always showed an "catkin_mil_ws", I'm sure that was intended
```

Comment:
I found an issue with the number of dashes being printed after changing the terminal's width, but nothing weird with respect to tmux. Can you explain the issue in a bit more detail?